CMakeLists.txt

```diff
@@ -184,6 +184,10 @@ install(TARGETS rmm
 
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/rmm
         DESTINATION include)
+
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/include/rmm
+        DESTINATION include)
```

Comment:
Is this just to get `cnmem.h` installed to be found when building against rmm? cc @harrism extra motivation to rip out cnmem 😄

Reply:
We plan to deprecate cnmem with the 0.15 release and remove it in 0.16 if possible. Then RMM will be header-only. #408

Reply:
Yes, just as there was an existing copy of `cnmem.h` to get it into the right include directory, this does the same but doesn't modify the source tree in the process. It's a relatively minor change, but I guess everyone will be happy when cnmem will be gone and such hacks aren't required anymore.

It's not required for what I'm doing, so I can keep it as-is. But actually my first interaction with RMM was that I asked a student to try using it, and he messed something when trying to install it, and ended up with a `rmm/detail/cnmem.h` include not found error, and that was confusing, because in fact no such file existed in the source tree to start with. And when looking at what happened, I didn't expect that the build would modify the source tree.